mappings/net/minecraft/network/packet/s2c/play/BlockEventOccurredS2CPacket.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_azqekdgf net/minecraft/network/packet/s2c/play/BlockEventS2CPacket
+CLASS net/minecraft/unmapped/C_azqekdgf net/minecraft/network/packet/s2c/play/BlockEventOccurredS2CPacket
```

Comment:
How about `BlockEventHappened`?

Reply:
I like that, `happened` is a more commonly used word that `occurred`

Reply:
however, I think what I'm going to go with is no verb on event packets
they read fine without it